stylesheet.css

```diff
@@ -1,6 +1,7 @@
 * {
   box-sizing: border-box;
   background: #e5e5e5;
+  margin: 0;
 }
 
 button:hover {
```

Comment:
:red_circle: You have done well implementing the hover effect :clap: :100: . Kindly make sure your hover state reflects the Figma Design  as shown in the screenshot below :arrow_down: 

![buuton](https://user-images.githubusercontent.com/26736582/199902936-c306e3b1-2d7a-47e3-895b-f63620402797.png)